### DISCUSSION TITLE
Help: Logs shown in Elasticsearch, but not Kibana

### DISCUSSION THREAD

**Discussion by cyberphor** [Security-Onion-Solutions/securityonion#3386] [Unsupported Versions] [answered]
VERSION: 2.3.2
INSTALL SOURCE: ISO image
INSTALL TYPE: Distributed, Air-gap; Manager + Search node
STATUS: All services say OK
SALT-CALL: No failures
ISSUE: I'm not able to view logs via Kibana even though I'm able to query them in Elasticsearch. They only show-up when I create a new Index Pattern without a time filter. The specific logs I'm focused on are Windows Events being shipped via Winlogbeat.

**Comment by weslambert** [comment, marked as answer]
Have you tried refreshing the existing index patterns?  If you exclude the logs, by creating a more narrowly-focused index pattern (like `*:so-zeek*`), are you able to view the logs as you normally would?

**Reply by cyberphor** [reply]
Sorry for the late reply. Yes, I tried that first. It appears all is well now. My fix-action was correcting the local time of my Analyst Workstation. It was several hours behind the Manager Node. A good lesson to learn though. I'll be continuously checking not only time across my network, but specifically the time of my Analyst Workstation. Again, this was all on an Air Gap network.
